### PR TITLE
Update roadmap.md

### PR DESCRIPTION
**Pull request by saulmc**
-keystore to GA
-react to DevPre
-update react native link

### CODE REVIEW

File: vision/roadmap.md [hunk 1]
```diff
@@ -71,14 +63,18 @@ A super early peek into what we're building. Ready for exploration, not adoption
...
 
 ---
 
 ## Developer Preview 🧑‍💻
 
 Opportunity to test, validate, and provide feedback. We do not recommend usage in production apps just yet.
 
+### UI Kit & React Hooks SDK
+
+Enable developers to focus on differentiating features by providing reusable messaging components via a [UI kit & React Hooks SDK](https://github.com/xmtp/xmtp-web).
```
Comment (jhaaaa) [inline vision/roadmap.md:76]: I'd suggest linking directly to the React SDK directory in the monorepo:

```suggestion
Enable developers to focus on differentiating features by providing reusable messaging components via a [UI kit & React Hooks SDK](https://github.com/xmtp/xmtp-web/tree/main/packages/react-sdk).
```

**Review by jhaaaa — APPROVED** [review]
Thank you for this @saulmc! Added a tiny suggestion...

File: vision/roadmap.md [hunk 2]
```diff
@@ -105,6 +101,10 @@ Developers are able to build iOS apps in Swift using the [`xmtp-ios` SDK](/docs/
 
 Developers are able to build Flutter and mobile apps in Dart using the [`xmtp-flutter` SDK](/docs/client-sdk/dart/tutorials/quickstart).
 
+### Secure key storage module
+
```
Comment (jhaaaa) [inline vision/roadmap.md:105]: @saulmc - With v8, I believe the Keystore exists in app memory only. From what I understand from @neekolas, the next step is to allow it to exist in other more secure contexts, such as wallet apps.

Comment (neekolas) [reply vision/roadmap.md:105]: It lives in the app, but not just in memory. We make use of the app's LocalStorage which is persistent.

**Pull request merged**